avaframe/com1DFA/particleInitialisation.py

```diff
@@ -0,0 +1,246 @@
+"""
```

Comment:
there are no pytests on any functions, is this on purpose?

Reply:
now added